tests/test_scripts.py

```diff
@@ -2777,6 +2776,47 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting_load_gzipped_bigcount():
+    infile = utils.get_temp_filename('test_ct')
+    ct = khmer.new_counting_hash(10, 1e7, 4)
```

Comment:
you can decrease this a lot, which is probably for the best -- maybe use 1e5 instead of 1e7?